GitCommands/Git/GPGLine.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace GitCommands.Gpg
+{
+    internal sealed class GPGLine // : IEquatable<GPGLine>
```

Comment:
Does it need to be a `class`? Can it be a struct? A readonly struct?

Also, remove commented out code.